Resources/Prototypes/Entities/Clothing/Masks/masks.yml

```diff
@@ -115,6 +116,7 @@
         Slash: 0.90
         Piercing: 0.95
         Heat: 0.95
+  - type: FilterMask
```

Comment:
just put it on gas mask since it should filter air anyway all 3 of these inherit from it anyway